docker/Dockerfile

```diff
@@ -17,27 +15,21 @@ RUN apt-get update && \
         zlib1g-dev \
         libncurses5-dev \
         libffi-dev \
-        libgdbm-dev && \
-   apt-get install -y default-mysql-server
-
-# Install nodejs and imagemagick
-WORKDIR /opt
-RUN wget https://nodejs.org/dist/v12.18.3/node-v12.18.3-linux-x64.tar.xz && \
-    tar xf node-v12.18.3-linux-x64.tar.xz && \
-    wget https://imagemagick.org/archive/binaries/magick && \
-    chmod +x magick && \
-    mv magick /usr/local/bin/magick
-
-ENV NODEJS_HOME=/opt/node-v12.18.3-linux-x64/bin
-ENV PATH=$NODEJS_HOME:$PATH
+        libgdbm-dev \
+        nodejs \
+        node-imagemagick \
+        imagemagick
 
 # Add core code to container
 WORKDIR /code
-COPY . /code
+COPY Gemfile Gemfile.lock /code/
 RUN gem install bundler -v 2.4.22 && \
     bundle install
 
+COPY . /code
+RUN bundle exec rails assets:precompile && \
```

Comment:
NB!
---
If anyone decides to port this change, please note that we do not have (yet) a separate Dockerfile for development _and_ precompiled assets are not excluded from git tracking via `.gitignore`, meaning that without changing the above, doing so will heavily pollute the working directory.

Reply:
Partially outdated due to #1430 - we now allow custom Dockerfiles to be used so as users can choose what's best for them. Still haven't excluded precompiled assets from tracking, though.